### ISSUE TITLE
Mastodon.trending_statuses and Mastodon.trending_links Each Return One Item

### ISSUE THREAD

**Issue by pdeitel** [halcy/Mastodon.py#338]
Hi, I was trying `Mastodon.trending_statuses` and `Mastodon.trending_links` methods. 

The docs for each say these methods can receive the `limit` parameter to specify the number of items to return. Both fail with 

      `TypeError: got an unexpected keyword argument 'limit'`

when called with a `limit` argument and both return a `dict`, not an "Array of" objects, as specified in the Mastodon API.

**Comment by halcy** [comment]
yep, the parameters are just... missing. Oops. As for return values: huh. Will have to check that.

**Comment by halcy** [comment]
Missing parameters fixed in master, rest to be investigated.

**Comment by halcy** [comment]
seems to work as it should in master ver.
